### PR TITLE
add first draft for SectionResponse with Stages

### PR DESCRIPTION
**Pull request by harryurban**
as roughly discussed here (desktop viewport), see second comment with enhanced model with config

Screenshot 2020-04-22 at 15 46 34

model shown in this image has no configs, only articles

### CODE REVIEW

**Comment by harryurban** [comment]
Visualize commit `add config for linkable label above a stage/group`
SectionResponse

File: proto/section_response.proto [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+message SectionResponse {
+  repeated Stage stages = 1;
+}
+
+message Stage {
+  ArticleGroup main = 1;
+  ArticleGroup sib = 2;
+}
+
+message ArticleGroup {
```
Comment (bobaaaaa) [inline proto/section_response.proto:17]: IMO this is a `Stage`

File: proto/section_response.proto [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import "content.proto";
+
+option java_package = "de.stroeer.model.wrapper.generated";
+option java_multiple_files = true;
+
+message SectionResponse {
+  repeated Stage stages = 1;
+}
+
+message Stage {
```
Comment (bobaaaaa) [inline proto/section_response.proto:12]: `Section` maybe better?

File: proto/section_response.proto [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  ArticleGroup main = 1;
+  ArticleGroup sib = 2;
+}
+
+message ArticleGroup {
+  repeated Article articles = 1;
+  GroupConfig config = 2;
+}
+
+message GroupConfig {
```
Comment (bobaaaaa) [inline proto/section_response.proto:22]: Actually this is a `StageConfig`

File: proto/section_response.proto (hunk 1 above)
Comment (bobaaaaa) [inline proto/section_response.proto:17]: I don't like this name, but I don't have a better one. What happened to the idea to introduce a `Teaser` model on `Section`'s?

File: proto/section_response.proto [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+option java_package = "de.stroeer.model.wrapper.generated";
+option java_multiple_files = true;
+
+message SectionResponse {
+  repeated Stage stages = 1;
+}
+
+message Stage {
+  ArticleGroup main = 1;
+  ArticleGroup sib = 2;
```
Comment (bobaaaaa) [inline proto/section_response.proto:14]:
```suggestion
  repeated ArticleGroup groups = 1;
```

Does this work too?

File: proto/section_response.proto (hunk 2 above)
Comment (bobaaaaa) [inline proto/section_response.proto:12]: IMO: The whole meaning of a `Stage` is now lost. Maybe we reverse the naming a bit:

```proto
message Group {
  Stage main = 1
}

message Stage {
  repeated Teaser content = 1
  StageConfig config = 2
}

message StageConfig {
 string label = 1
}
```

**Comment by harryurban** [comment]
thanks for the feedback. lets focus on #4

**Pull request closed without merging**